sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-b2c-oidc/pom.xml

```diff
@@ -50,6 +50,11 @@
       <artifactId>spring-boot-starter-security</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Why are we adding this here? Should we also delete some dependencies at the same time?

Reply:
There's no need to delete at the same time because I just make oauth2 client dependencies as an optional selection. When the scenario is a resource server app, we should remove the dependency.